### ISSUE TITLE
Stuck at "Initialising" after 3.0.3.(9) Hossa update

### ISSUE THREAD

**Issue by nephros** [mentaljam/harbour-storeman#94]
Storeman does not initalize properly since I updated to 3.0.3.9  on Jolla1.

The pulley on the main menu stays in "Initialising" state.

The "Recently updated", "Categories" "Bookmarks" and "Local RPM Files" sections are active and can be accessed, "Repositories" and "Installed Applications" can not.

In an app screen, installstatus is not shown correctly (everything is "not installed"), and there is no top pulley menu.

```
[nemo@Sailfish nemo]$ /usr/bin/harbour-storeman
[D] unknown:0 - Using Wayland-EGL
[W] unknown:0 - Could not find any zN.M subdirs!
[W] unknown:0 - Theme dir "/usr/share/themes/jolla-ambient/meegotouch/z1.0/" does not exist
[C] unknown:0 - Could not read "/home/.zypp-cache/solv/@System/solv"

```

journalctl has this:

```
May 09 12:47:12 Sailfish lipstick[1556]: invoker: Invoking execution: '/usr/bin/harbour-storeman'
May 09 12:47:20 Sailfish dbus[787]: [system] Activating via systemd: service name='org.nemo.ssu' unit='dbus-org.nemo.ssu.service'
May 09 12:47:20 Sailfish systemd[1]: Starting SSU service...
May 09 12:47:21 Sailfish dbus[787]: [system] Successfully activated service 'org.nemo.ssu'
May 09 12:47:21 Sailfish systemd[1]: Started SSU service.
May 09 12:47:22 Sailfish packagekitd[6904]: Setting target on init: /var/cache/zypp
May 09 12:47:22 Sailfish invoker[1673]: WARNING: An inactive plugin is misbehaving - tried to show a window!
May 09 12:47:22 Sailfish invoker[1673]: WARNING: requestActivate() called for  QQuickView(0x134bd78)  which has Qt::WindowDoesNotAcceptFocus set.
May 09 12:47:23 Sailfish lipstick[1556]: [D] onCompleted:263 - coverActionIndicators created <unknown>
May 09 12:47:24 Sailfish packagekitd[6904]: openrepos-thebootroo is not cached! Do a refresh
May 09 12:47:25 Sailfish PackageKit[6904]: get-updates transaction /52804_debacecc from uid 100000 finished with success after 3221ms
```

````
[root@Sailfish nemo]# pkcon search storeman
Searching by details
Starting
Refreshing software list
Querying
Finished
Installed       harbour-storeman-0.1.5-1.armv7hl (installed)                    OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.22-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.29-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.28-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.1.1-1.armv7hl (openrepos-osetr)              OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.1.0-1.armv7hl (openrepos-osetr)              OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.26-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.24-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.21-2.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.25-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.30-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.23-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.33-3.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.31-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.32-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.1.4-1.armv7hl (openrepos-osetr)              OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.0.27-1.armv7hl (openrepos-osetr)             OpenRepos Client for Sailfish OS
Available       harbour-storeman-0.1.1-2.armv7hl (openrepos-osetr)              OpenRepos Client for Sailfish OS
Available       harbour-storeman-turing-0.0.26-1.armv7hl (openrepos-osetr)      OpenRepos Client for Turing Phone
```

**Comment by nephros** [comment]
Note that this particular phone did NOT update to 3.0.3 without problems.

I had to recover manually by removing and reinstalling some packages (actually I just reinstalled all).

It's possible something in the failing upgrade process is the cause of this, e.g. `/home/.zypp-cache` does not exist on my system.

**Comment by nephros** [comment]
Found it. Borked system. Wrong/old versions were installed. Sorry for the noise.

Before:

``` 
[root@Sailfish nemo]# pkcon search PackageKit
Searching by details
Starting
Refreshing software list
Querying
Finished
Installed       PackageKit-1.1.9-1.3.3.jolla.armv7hl (installed)                Package management service
Available       PackageKit-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)               Package management service
Installed       PackageKit-Qt5-0.9.6+git-1.3.1.jolla.armv7hl (installed)        Qt support library for PackageKit
Available       PackageKit-Qt5-debuginfo-0.9.6+git-1.3.1.jolla.armv7hl (jolla)  Debug information for package PackageKit-Qt5
Available       PackageKit-Qt5-debugsource-0.9.6+git-1.3.1.jolla.armv7hl (jolla)        Debug sources for package PackageKit-Qt5
Available       PackageKit-Qt5-devel-0.9.6+git-1.3.1.jolla.armv7hl (jolla)      Development headers for PackageKit-Qt
Available       PackageKit-command-not-found-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)     Ask the user to install command line programs automatically
Available       PackageKit-debuginfo-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)     Debug information for package PackageKit
Available       PackageKit-debugsource-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)   Debug sources for package PackageKit
Available       PackageKit-doc-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)           Documentation for PackageKit
Installed       PackageKit-glib-1.1.9-1.3.3.jolla.armv7hl (installed)           GLib libraries for accessing PackageKit
Available       PackageKit-glib-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)          GLib libraries for accessing PackageKit
Available       PackageKit-glib-devel-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)    GLib Libraries and headers for PackageKit
Installed       PackageKit-zypp-1.1.9-1.3.3.jolla.armv7hl (installed)           PackageKit zypp backend
Available       PackageKit-zypp-1.1.9+git5-1.8.1.jolla.armv7hl (jolla)          PackageKit zypp backend
``` 

Fix it: 

``` 
[root@Sailfish Downloads]# pkcon update PackageKit-zypp
Resolving
Querying
Testing changes
Finished                                                                                                                                               [                                                                           ] (0%)
The following packages have to be installed:
 PackageKit-1.1.9+git5-1.8.1.jolla.armv7hl      Package management service
 PackageKit-glib-1.1.9+git5-1.8.1.jolla.armv7hl GLib libraries for accessing PackageKit
 PackageKit-zypp-1.1.9+git5-1.8.1.jolla.armv7hl PackageKit zypp backend
 boost-thread-1.66.0-1.3.8.jolla.armv7hl        Run-Time component of boost thread library
 libsolv-tools-0.6.35+git2-1.4.5.jolla.armv7hl  A new approach to package dependency solving
 libsolv0-0.6.35+git2-1.4.5.jolla.armv7hl       A new approach to package dependency solving
 libzypp-17.3.1+git4-1.6.2.jolla.armv7hl        Package, Patch, Pattern, and Product Management
 ssu-0.43.12-1.8.1.jolla.armv7hl        SSU enabler for RND
Proceed with changes? [N/y] y
```
